gtk4/src/closure_expression.rs

```diff
@@ -23,16 +23,13 @@ define_expression!(
 
 impl ClosureExpression {
     #[doc(alias = "gtk_closure_expression_new")]
-    pub fn new<F, R>(params: impl IntoIterator<Item = impl AsRef<Expression>>, callback: F) -> Self
+    pub fn new<R, I, E>(params: I, closure: glib::Closure) -> Self
```

Comment:
Make it take a RustClosure ?

Reply:
It probably has to for type-safety reasons anyway.

Reply:
I changed it to `RustClosure`, if someone wants to add another function that takes just `Closure` it should probably be unsafe anyway.